atmo/clusters/provisioners.py

```diff
@@ -7,10 +7,25 @@
 class ClusterProvisioner(Provisioner):
     log_dir = 'clusters'
 
+    def __init__(self):
+        super(ClusterProvisioner, self).__init__()
+        # the S3 URI to the script-runner jar
+        self.jar_uri = (
+            's3://%s.elasticmapreduce/libs/script-runner/script-runner.jar' %
+            self.config['AWS_REGION']
+        )
```

Comment:
There is also a `jar_uri` attribute in the `atmo.jobs.provisioners.SparkJobProvisioner`. Best would be to move this to the base class `atmo.provisioners.Provisioner` and remove it from the jobs and cluster provisioners. 🦆